plugins/apps/threatbus_misp/plugin.py

```diff
@@ -200,7 +201,24 @@ def run(
         validate_config(config)
     except Exception as e:
         logger.fatal("Invalid config for plugin {}: {}".format(plugin_name, str(e)))
-    if config["api"].get():
+
+    filter_config = config["filter"].get(list)
+
+    # start Attribute-update receiver
+    if config["zmq"].get():
+        receiver_thread = threading.Thread(
+            target=receive_zmq, args=(config["zmq"], inq), daemon=True
+        )
+    elif config["kafka"].get():
+        receiver_thread = threading.Thread(
+            target=receive_kafka, args=(config["kafka"], inq), daemon=True
+        )
```

Comment:
`else` fail if neither is found?

Reply:
No, it should be allowed to not receive intel from MISP (e.g., the user just wants to send back sightings).
I made this more explicit (see latest commit).